applications/Shopbring.md

```diff
@@ -0,0 +1,320 @@
+
+Our team also uses the [openwallet developer platform](https://www.openwallet.cn) to provide enterprises with blockchain-related development services, such as: cryptocurrency wallets, enterprise-level cryptocurrency management systems, multi-chain multisig wallet systems.
+
+### Team Code Repos
+
+* https://github.com/blocktree
+* https://github.com/zhiquan911
+* https://github.com/godaddy-x
+* https://github.com/fanyinghao
+* https://github.com/Brochao
+
+### Team LinkedIn Profiles
+
+* https://www.linkedin.com/in/zhiquan911
+* https://www.linkedin.com/in/shadowsick
+* https://www.linkedin.com/in/kencani
+* https://www.linkedin.com/in/fanyinghao
+* https://www.linkedin.com/in/heshuchao
+* https://www.linkedin.com/in/maerfeng
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+
+* **Total Estimated Duration:** 3 months.
+* **Full-time equivalent (FTE):** 5 FTE.
+* **Total Costs:** 23K DAI.
+
+### Milestone 1 UI design and System module definition
+
+* **Estimated Duration:** 1.5 month
+* **FTE:**  2
+* **Costs:** 11K DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | We write complete program flow description and system module interface definition. |
+| 0c. | Testing Guide | We will deploy a POA consensus Shopbring network, which can call system modules through polkadot.js. |
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. |
+| 1. | Substrate module: Generic Asset | Programming language: `Rust`. This module aims to manage multiple cross-chain assets. It has the following functions: `register_generic_assets`, `deregister_generic_assets`, `mint_generic_assets`, `burn_generic_assets`, `transfer_assets`. |
+| 2. | Substrate module: Invitation | Programming language: `Rust`. This module aims to establish the invitation relationship between accounts. It has the following functions: `register_inviter`, `accept_invitation`, `end_invitation_period`. |
+| 3. | Substrate module: Commissioned Shopping | Programming language: `Rust`. This module aims to implement the commissioned shopping process. It has the following functions: `apply_shopping_order`, `accept_shopping_order`, `do_commodity_shipping`, `confirm_commodity_received`, `apply_commodity_return`, `accept_commodity_return`, `do_commodity_returning`, `confirm_commodity_returned`. |
+| 4. | Substrate module: Transaction Payment | Programming language: `Rust`. This module aims to rewrite the `pallet_transaction_payment` so that it can change the payment `origin` of the transaction fee according to the specific module's functions.|
```

Comment:
Does it mean you're planning to fork and maintain your own pallet, or rather do you aim to submit a PR to merge these changes upstream? With the details provided, it is difficult to judge whether one or the other option is preferable - though generally maintaining your own fork is discouraged.

Reply:
In the early stage, we intend to fork and maintain our own pallet, which can avoid some version conflicts. When we are free in later work, we will submit a PR to merge these changes upstream.